maestro/src/main/java/org/intocps/maestro/cli/ImportCmd.java

```diff
@@ -46,9 +45,8 @@
     boolean preserveAnnotations;
     @CommandLine.Option(names = {"-if", "--inline-framework-config"}, description = "Inline all framework configs", negatable = true)
     boolean inlineFrameworkConfig;
-    @CommandLine.Option(names = {"-fmu", "--fmu-base-dir"},
-            description = "A path to the FMU base directory at which the FMU parhs are " + "relative to")
-    File fmuBase;
+    @CommandLine.Option(names = {"-fsp", "--fmu-search-path"}, description = "A search path used to resolve relative FMU paths.")
```

Comment:
Måske "One or more search paths used to resolve relative FMU paths. "
Hvordan skal argumentet gives? `-fsp sti1 sti2 sti3 eller` `-fsp sti1 -fsp sti2`